Cargo.toml

```
@@ -23,6 +23,9 @@ zeroize = { version = "1", default-features = false, features = ["derive"] }
 rkyv = { version = "0.7", optional = true, default-features = false }
 bytecheck = { version = "0.6", optional = true, default-features = false }
 rayon = { version = "1.8", optional = true }
+serde = { version = "1.0.210", optional = true }
+bs58 = { version = "0.4.0" , optional = true }
+serde_json = { version = "1.0.128", optional = true }
```

```suggestion
serde = { version = "1.0", optional = true }
bs58 = { version = "0.4" , optional = true }
serde_json = { version = "1.0", optional = true }
```

No need to specify the version for such a granular subversion